man/systemctl.xml

```diff
@@ -1690,111 +1690,114 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
           <term><command>default</command></term>
 
           <listitem>
-            <para>Enter default mode. This is mostly equivalent to
-            <command>isolate default.target</command>.</para>
+            <para>Enter default mode. This is mostly equivalent to <command>systemctl isolate
+            default.target</command>. This operation is blocking by default, use <option>--no-block</option> to request
+            asynchronous behavior.</para>
           </listitem>
         </varlistentry>
 
         <varlistentry>
           <term><command>rescue</command></term>
 
           <listitem>
-            <para>Enter rescue mode. This is mostly equivalent to
-            <command>isolate rescue.target</command>, but also prints a
-            wall message to all users.</para>
+            <para>Enter rescue mode. This is mostly equivalent to <command>systemctl isolate rescue.target</command>,
+            but also prints a wall message to all users. This operation is blocking by default, use
```

Comment:
AFAIK systemd doesn't implement a wall message anywhere for rescue / emergency mode.  The wall message is implemented thru logind only.  We don't use logind for rescue / emergency.

Then "mostly equivalent" can probably be replaced with "equivalent" for these two verbs.

Reply:
indeed